src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -49,5 +49,20 @@ static constexpr Property<bool> denormals_optimization{"CPU_DENORMALS_OPTIMIZATI
 
 static constexpr Property<float> sparse_weights_decompression_rate{"SPARSE_WEIGHTS_DECOMPRESSION_RATE"};
 
+/**
+ * @brief This property define if using hyper threading for CPU inference.
+ * @ingroup ov_runtime_cpu_prop_cpp_api
+ *
+ * Developer can use this property to use or not use hyper threading for CPU inference. If user does not explicitly set
+ * value for this property, OpenVINO may choose any desired value based on internal logic.
+ *
+ * The following code is example to use this property.
+ *
+ * @code
+ * ie.set_property(ov::intel_cpu::use_hyper_threading(true));
+ * ie.set_property(ov::intel_cpu::use_hyper_threading(false));
```

Comment:
1. is it valid to set this property globally w/o device name?
2. should it also be hint because `use_hyper_threading(true)` does not guarantee performance difference on processors without HT?

Reply:
No. it is only intel_cpu property now. 

Would you like to extend to ov property for all other plugs? If yes, we can move to ov::hint